app/_how-tos/kic-vault-hashicorp.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Configure Hashicorp Vault
+description: "Configure Hashicorp Vault with {{ site.kic_product_name }} and the KongVault CRD"
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/vault/hashicorp/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - /gateway/secrets-management/
+
+min_version:
+  kic: '3.1'
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I configure Hashicorp Vault with {{ site.kic_product_name }}?
+  a: Create a `KongVault` CRD and then use the `vault://` reference in your plugin configuration
+
+prereqs:
+  enterprise: true
+  kubernetes:
+    skip_proxy_ip: true
+  inline: 
+    - title: HashiCorp Vault
+      include_content: prereqs/hashicorp-vault-k8s
+      icon_url: /assets/icons/hashicorp.svg
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kubernetes.svg
+    - title: Uninstall HashiCorp Vault from your cluster
+      content: |
+        ```bash
+        kubectl delete namespace vault
+        ```
+      icon_url: /assets/icons/hashicorp.svg
+
+next_steps:
+  - text: Review the Vaults entity
+    url: /gateway/entities/vault/
+
+related_resources:
+  - text: Secrets management
+    url: /gateway/secrets-management/
+
+tags:
+  - secrets-management
+  - security
+
+---
+
+## Create a KongVault entity
+
+{{ site.kic_product_name }} uses the `KongVault` entity to configure the connection to a Vault. As we're running Hashicorp Vault in `dev` mode, we can use the `root` token to access the Vault:
```

Comment:
When I ran this, I got:
```
error: resource mapping not found for name: "hcv" namespace: "kong" from "STDIN": no matches for kind "KongVault" in version "configuration.konghq.com/v1"
ensure CRDs are installed first
```

I tried using the [CRD prereq](https://deploy-preview-1310--kongdeveloper.netlify.app/kubernetes-ingress-controller/rate-limiting/#enable-the-gateway-api) from another how to, but that didn't solve it.